west.yml

```diff
@@ -8,14 +8,11 @@ manifest:
   remotes:
     - name: zephyrproject-rtos
       url-base: https://github.com/zephyrproject-rtos
-    - name: u-blox
```

Comment:
🦀u-blox is gone 🦀

Reply:
More like under new management
![image](https://github.com/user-attachments/assets/968a8d4f-328a-4ec1-988a-9e65e8c581ef)